fs/move.py

```diff
@@ -53,26 +58,55 @@ def move_file(
         dst_path (str): Path to a file on ``dst_fs``.
         preserve_time (bool): If `True`, try to preserve mtime of the
             resources (defaults to `False`).
+        cleanup_dst_on_error (bool): If `True`, tries to delete the file copied to
+            `dst_fs` if deleting the file from `src_fs` fails (defaults to `True`).
```

Comment:
Looking at the help-string above for the `src_path` and `dst_path` parameters, should the `dst_fs` and `src_fs` here be in **double** backticks? (see https://docs.pyfilesystem.org/en/latest/reference/move.html#fs.move.move_file )